### ISSUE TITLE
ES modules

### ISSUE THREAD

**Issue by zigomir** [WebReflection/hyperHTML#102]
Now that Safari and Chrome already support ES modules natively, it would be nice if one could import this library like so

```js
import hyper, { wire } from 'https://unpkg.com/hyperhtml@latest/min.js'
```

I'm aware that current setup with uglify-js doesn't support ES6 syntax yet. There might be other issues too. I'm mostly curious if this is something you'd consider adding or it isn't something worth adding or not really?

**Comment by WebReflection** [comment]
you can see exports here:
https://github.com/WebReflection/hyperHTML/commit/1e81342ae68b4a959c78be964db652df18adc860#diff-39e956f0c2d414da37239d9a1bce0f3aR4

and you can test the browser here:
https://webreflection.github.io/hyperHTML/test/mjs.html

I need to double check about **unpkg** because I also use their cdn a lot but it's apparently not serving the correct mime-type like GitHub does.

TL;DR the package is the exact same but it now contains the ugly _.mjs_ files since this is apparently how things are done these days.

**Comment by WebReflection** [comment]
Follow up as PR:
https://github.com/unpkg/unpkg-website/pull/65

**Comment by zigomir** [comment]
Ah, I see now, thanks!

**Comment by WebReflection** [comment]
> oh wow, that was actually done as new version

Oh yes, I forgot to mention I [pushed changes](https://github.com/WebReflection/hyperHTML/commit/1e81342ae68b4a959c78be964db652df18adc860) to make your request happen.
